pkg/sql/parser/var_name.go

```diff
@@ -82,10 +82,7 @@ var _ VarName = &ColumnItem{}
 // NormalizeVarName is a no-op for UnqualifiedStar (already normalized)
 func (u UnqualifiedStar) NormalizeVarName() (VarName, error) { return u, nil }
 
-var singletonStarName VarName = UnqualifiedStar{}
-
-// StarExpr is a convenience function that represents an unqualified "*".
-func StarExpr() VarName { return singletonStarName }
+var UnqualifiedStarInstance = UnqualifiedStar{}
```

Comment:
Move this to `name_part.go` near the type definition.

Reply:
Done.